README.md

```diff
@@ -1,5 +1,7 @@
 # Amazon Corretto Crypto Provider
-The Amazon Corretto Crypto Provider (ACCP) is a collection of high-performance cryptographic implementations exposed via the standard [JCA/JCE](https://docs.oracle.com/en/java/javase/11/security/java-cryptography-architecture-jca-reference-guide.html) interfaces. This means that it can be used as a drop in replacement for many different Java applications. Currently algorithms are primarily backed by OpenSSL's implementations (1.1.1d) but this may change in the future.
+The Amazon Corretto Crypto Provider (ACCP) is a collection of high-performance cryptographic implementations exposed via the standard [JCA/JCE](https://docs.oracle.com/en/java/javase/11/security/java-cryptography-architecture-jca-reference-guide.html) interfaces.
+This means that it can be used as a drop in replacement for many different Java applications.
+Currently algorithms are primarily backed by OpenSSL's implementations (1.1.1f as of ACCP 1.4.0) but this may change in the future.
```

Comment:
* The cryptographic algorithms are backed by OpenSSL 1.1.1f as of ACCP 1.4.0, but this is subject to change.

Reply:
We implement HMAC and AES-CTR-DRBG ourselves (though backed by openssl primitives). So, while this language is a bit longer, I feel it is more accurate, and so will keep it.